### DISCUSSION TITLE
Attention message from SOLWEIG

### DISCUSSION THREAD

**Discussion by SimoneBeltramino** [UMEP-dev/UMEP#397] [Q&A]
Trying to make a simulation of outdoor thermal comfort with SOLWEIG returns the following error message: "The land cover grid includes integer values higher (or lower) than UMEP-formatted land cover grid (should be integer between 1 and 7). If other LC-classes should be included they also need to be included in landcoverclasses_2016a.txt."  I have fixed the cell size and checked that the values are integers. How can I fix this error message?

 - QGIS Version [3.24]
 - UMEP Version [e.g. 3.19.3]
<img width="913" alt="Immagine 2022-04-14 174650" src="https://user-images.githubusercontent.com/103640974/163426659-83d9deec-6462-4b21-a92b-1ac1d909cd25.png">
<img width="468" alt="Immagine1" src="https://user-images.githubusercontent.com/103640974/163426691-ae1ac3a5-324c-4293-88bd-4b8bf9dba39f.png">

Thank you so much!

**Comment by biglimp** [comment]
In Solweig, ground cover should be represented, i.e. What is present underneath trees and bushes. Therefore the two tree classes should not be present. Also, you can have no Nodata-values in your LC-data. Reclassify to e.g. 1.
